engine/src/test/java/org/pentaho/di/www/ExecuteTransServletTest.java

```diff
@@ -184,7 +185,7 @@ public void doGetTransformationNotFoundTest() throws Exception {
     HttpServletRequest mockHttpServletRequest = mock( HttpServletRequest.class );
     HttpServletResponse mockHttpServletResponse = mock( HttpServletResponse.class );
     Trans trans = initMocksForTransExecution( mockHttpServletRequest, mockHttpServletResponse );
-    doThrow( new KettleException( "Unable to find transformation" ) ).when( executeTransServlet ).executeTrans( trans );
+    doThrow( new KettleException( "Unable to find transformation" ) ).when( executeTransServlet ).executeTrans( trans, new HashMap<>());
```

Comment:
i think better to override this method for our approach and handle it java rather than doing it in test class